### DISCUSSION TITLE
Angular with WSL2

### DISCUSSION THREAD

**Discussion by gtmsingh** [alkampfergit/personal-blog#7] [Q&A]
I was excited about using WSL2 with angular (as [was the author](https://www.codewrecks.com/post/general/compile-angular-in-wsl2/)) but soon enough I was stuck with WSL2 connection being broken as soon as I compile my angular project.

Does this happen with others too? I am not sure if I should raise a bug / fetch details for this issue to be able to raise bug.

**Comment by alkampfergit** [comment]
I think that everything will be more easy with windows 11, where you can run GUI in WSL2. Actually I'm running Code or WebStorm directly in WSL (so I have no connection problem) in a Windows 11 machine.

**Reply by gtmsingh** [reply]
@alkampfergit that sounds promising. Could you also confirm if Win 11 WSL2 is still stable when using after-save hooks to reformat files automatically? I found that if I turn off this hook, the editor works just fine in Win 10.

**Reply by alkampfergit** [reply]
It seems stable, I did not use windows 11 as my primary development machine, but I played around with python a little bit with pycharm and didn't have problems.

**Reply by gtmsingh** [reply]
Got it, thanks! I will keep this open in case someone can confirm it. In the mean time, I will try to see better way to have these hooks configured.
